mintapi/signIn.py

```diff
@@ -75,8 +71,7 @@ def get_email_code(
                 break
             rv, data = imap_client.fetch(num, "(RFC822)")
             if rv != "OK":
-                logger.error("ERROR: ERROR getting message", num)
-                sys.exit(1)
+                raise RuntimeError("Unable to complete due to error message " + num)
```

Comment:
Is `num` actually meaningful to the user? Would `rv` be better here? I'm not actually sure what it looks like when it isn't "OK" or if `data` is better.

Reply:
That's a good question. `rv` is most likely the most meaningful and we can standardize the inclusion of that in each message.